fcrepo-clients/src/test/java/edu/unc/lib/dl/fcrepo4/BinaryObjectTest.java

```diff
@@ -0,0 +1,128 @@
+package edu.unc.lib.dl.fcrepo4;
```

Comment:
You are correct. I've been working this morning on pulling out these files. I accidentally branched from my local fcrepo4 instead of the remote branch.